### ISSUE TITLE
WHI support

### ISSUE THREAD

**Issue by cwoac** [cwoac/TTANR#3]

**Comment by cwoac** [comment]
Sample deck: `http://deckbox.org/sets/1007109/export`
deck is of the form

```
2 Arcane Power
3 Ariel. Queen in the Woods
3 Contested Village
3 Dance to Loec
3 Forest Dragon
3 Forgotten Cairn
3 Fury of the Forest
3 Highborn Champion
3 Long Winter
3 Nimble Spearman
2 Pageant of Shrikes
3 Pilgrimage
3 Remote Monastery
3 Slumbering Titan
2 Spellweaver
1 Star Crown Fragments
2 Talismanic Tatoos
2 The Wild Hunt
3 Wildwood Grove
Sideboard:
```

Sample card url: `https://deckbox.org/whi/Ariel.%20Queen%20in%20the%20Woods`
card img link is of form `<img id="card_image" align="left" src="/system/images/whi/cards/42-10.jpg" alt="Ariel. Queen in the Woods" />`

**Comment by cwoac** [comment]
Fixed by 836f0f4a381a9acc195cc229fe7699312b300d99
